components/ide/jetbrains/backend-plugin/gradle-stable.properties

```diff
@@ -1,9 +1,9 @@
 # See https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
 # for insight into build numbers and IntelliJ Platform versions.
-pluginSinceBuild=222.4459
+pluginSinceBuild=222.4345
```

Comment:
It seems the JSON version of Snapshots Page has different order from the HTML one:
<img width="378" alt="image" src="https://user-images.githubusercontent.com/418083/201705282-191016d2-2617-496b-aeed-89926909c6b0.png">
Latest one is still 222.4459.
cc @andreafalzetti